data/formats-data.js

```diff
@@ -3690,7 +3697,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	chimchar: {
-		randomBattleMoves: ["stealthrock", "overheat", "hiddenpowergrass", "fakeout", "uturn", "gunkshot"],
+		randomBattleMoves: ["stealthrock", "fireblast", "swordsdance", "firepunch", "uturn", "gunkshot", "thunderpunch"],
```

Comment:
Fake Out

Reply:
but there's so many good moves here and competition for the slots are so fierce...idk if it stands out enough